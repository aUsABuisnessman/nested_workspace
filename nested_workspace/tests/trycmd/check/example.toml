args = ["check", "-vv"]

[bin]
name = "cargo"

[fs]
cwd = "../../../../example"
