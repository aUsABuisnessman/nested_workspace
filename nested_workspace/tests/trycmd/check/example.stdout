[nested_workspace_example 0.1.0] [..]
